index.js

```diff
@@ -1,13 +1,13 @@
 var mapSize = 10;
 var i, j;
 
-for(i = 0; i < mapSize; i++){
+for(i = 1; i <= mapSize; i++){
     var divRow = document.createElement("div");
-    divRow.className = "row row-" + i;
-    document.body.appendChild(divRow);
-    for (j = 0; j < mapSize; j++){
+    divRow.classList.add("row", "row-" + i);
+    mapWraper.appendChild(divRow);
```

Comment:
It's not efficient to append child to DOM element in each iteration. Please, during creation of grid, make sure that you interact with DOM tree only once. I mean interaction with mapWraper. Consider to use document fragment for it.
Visit [mdn](https://developer.mozilla.org/en-US/docs/Web/API/Document/createDocumentFragment) and [this blogpost](https://davidwalsh.name/documentfragment) for more information
Also, you have a type here. Two `p` in `wrapper`.